### ISSUE TITLE
Avatar is diffusing inside the wall 

### ISSUE THREAD

**Issue by Akhilesh88** [mozilla/hubs#6395]
**Description**
Avatar can passtrough the wall.

**Steps To Reproduce**
1. Go to https://hubs.mozilla.com/E4e8oLx/hubs-demo-promenade 
2. Click on the Hubs community Festival ,
3. Visit the room by cliking on the Hubs Community Festival Room.
4. Wait for Object model to be loaded.Once you are inside the Festival Area move little Further
5. You will see the Hubs community Festival Banner board.
6. Next move right side you will see the information  area.
7.Move little further to Next event "Coming Soon to Hubs " Event Area.
8. Next Event Area with speaker and mic
9. Now try to move into the blue wall on lef side you can enter easily
![how to reproduce the bug go inside this blue wall](https://github.com/mozilla/hubs/assets/40929298/00b65e99-0807-4933-9b16-7c34206c471b)
10. See error.

**Expected behavior**
Avatar Should not be allowed to pass through the wall.
![Avatar inside the Wall Diffusing 2nd screenshot](https://github.com/mozilla/hubs/assets/40929298/0ddadfdd-69fc-4024-a06c-c2fd5ccf6f76)
![Avatar inside the Wall Diffusing](https://github.com/mozilla/hubs/assets/40929298/3641caab-e683-4e4e-8199-9cee765d3be8)
![Event area with speaker and mic](https://github.com/mozilla/hubs/assets/40929298/86e75cff-1e7c-4821-9513-2a66fcd181c9)
![Event Area with speaker and Mic](https://github.com/mozilla/hubs/assets/40929298/8d81466c-5ecb-4f10-92eb-513c2882b0bd)

**Screenshots**
Screenshots attached.

**Hardware** 
 - Device: [ i5 gen 5 Dell laptop 8 Gb Ram & 512 SSD WD Green ]
 - OS: [ Windows 10 Pro]
 - Browser: [Google chrome Version 119.0.6045.200 (Official Build) (64-bit)]

.

**Comment by Exairnous** [comment]
Thank you for the bug report!

However, Hubs doesn't use physics and physics collisions to determine where and how you move, but rather uses a navigation mesh specific to each scene to control where your avatar can walk and this navigation mesh is created by the scene creator.  So this could be an issue with this specific scene (or not, depending on the creator's intent), but it isn't a bug in Hubs.

If you are already aware of this and are saying that this specific scene needs fixing (potentially because it's part of the demo rooms?), please confirm this and disregard the rest of my comment.

**Comment by Akhilesh88** [comment]
> Thank you for the bug report!
> 
> However, Hubs doesn't use physics and physics collisions to determine where and how you move, but rather uses a navigation mesh specific to each scene to control where your avatar can walk and this navigation mesh is created by the scene creator. So this could be an issue with this specific scene (or not, depending on the creator's intent), but it isn't a bug in Hubs.
> 
> If you are already aware of this and are saying that this specific scene needs fixing (potentially because it's part of the demo rooms?), please confirm this and disregard the rest of my comment.


Yes these issue was reproducible but if possible scene can be fixed of that Immersive space area .
